src/compass/utils/h5_helpers.py

```diff
@@ -374,15 +376,23 @@ def identity_to_h5group(dst_group, burst, cfg):
              'OGR compatible WKT representation of bounding polygon of the image',
              {'units':'degrees'}),
         Meta('mission_id', burst.platform_id, 'Mission identifier'),
+        Meta('processing_date_time', datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
```

Comment:
A nit pick / question here. `processing_date_time` seems to use a slightly different formatting to `TIME_STR_FMT`. Is there some specific needs for doing that? If not, I would suggest to use `TIME_STR_FMT` like  `zero_doppler_start_time` and `zero_doppler_end_time` below.

Reply:
No preference I will change it to TIME_STR_FMT. I have stolen this metadata entry from [RTC](https://github.com/opera-adt/RTC/blob/cc9bf9cb1fd54dc9a80c71a7cb2c240422fcd9fd/src/rtc/h5_prep.py#L311). You might want to check if you are consistent there as well